src/Rector/AbstractPHPUnitRector.php

```diff
@@ -2,21 +2,18 @@
 
 namespace Rector\Rector;
 
-use Nette\Utils\Strings;
 use PhpParser\Node;
-use Rector\Node\Attribute;
+use PHPUnit\Framework\TestCase;
+use Rector\NodeTypeResolver\NodeTypeResolver;
 
 abstract class AbstractPHPUnitRector extends AbstractRector
 {
-    /**
-     * Check if the file is a PHPUnit TestCase. By default, it should end with "Test", as it is the standard.
-     *
-     * @see https://phpunit.de/getting-started-with-phpunit.html
-     */
     protected function isInTestClass(Node $node): bool
     {
-        $className = (string) $node->getAttribute(Attribute::CLASS_NAME);
+        $nodeTypeResolver = new NodeTypeResolver();
```

Comment:
@TomasVotruba Should we revert 691eb10? Because doing this way, isn't working by our CS :(

Reply:
@TomasVotruba ping.

Is there a way to DI without `__construct`? If not, revert #300 will be the solution (unfortunately)

Reply:
I missed this, sorry.

You can inject dependency as in Abstract class: https://github.com/rectorphp/rector/blob/9f4ca8071a2c06aeaae336f84c7cf5293fc211db/src/Rector/AbstractRector.php#L20-L48

That's Symfony approach to this